### DISCUSSION TITLE
feat: Stats/Reports/analytics of triggered campaigns

### DISCUSSION THREAD

**Discussion by drexlma** [chatwoot/chatwoot#8468] [Feature Requests]
how often was such a camp. triggered.

triggert on 12.12.2022: 4 times

**Comment by sojan-official** [comment]
@drexlma Thanks for the feedback. We will prioritize this for our future releases.

**Reply by pranavrajs** [reply]
I'm moving these to discussions as we will not be prioritizing these immediately.
